### ISSUE TITLE
All other metadata on page is deleted when selecting a text option

### ISSUE THREAD

**Issue by Lunkster** [mProjectsCode/obsidian-meta-bind-plugin#25]
This is the plugin I so much wanted, but when I'm trying it out it deletes all other metadata in frontmatter when selecting a text

```meta-bind
INPUT[select(option(vitt),option(rött),option(rosé),option(mousserande)):typ]
```

It adds the metadatafield "typ", but delete earlier manually added metadata. Other metadata fields from the plugin stays however.

Bug or I do it wrong?

**Comment by Lunkster** [comment]
I might define the bug a bit more:
Manually added metadata fileds in page is actually not altered, except when addad by template.
So,
If I import metadata fields from a template and thereafter choose any metadata through the plugin-functions, the metadata is totally removed. Even MD-field previously added with the plugin are removed.

If I then add MD-fields manually and use the plugin, everything works.
So it related to the Templates core plugin

**Comment by mProjectsCode** [comment]
That is strange, I will test this later today. I have a suspicion that it has something to do with reading metadata from the file cache

**Comment by mProjectsCode** [comment]
this is also the same core issue as #20

**Comment by mProjectsCode** [comment]
Should be fixed in `0.4.0`
